README.md

```diff
@@ -205,7 +205,7 @@ By default, `gen-grpc-gateway` will discard unknown fields from requests. You ca
 
 ### Environment Variables
 
-The gateway project used [spf13/viper](https://github.com/spf13/viper) for configuration. The generated gateway code includes a config file that can be overridden with cli flags or environment variables. For environment variable overrides use a `<SERVICE>_` prefix, upcase the setting, and replace `.` with `_`.
+The gateway project used [spf13/viper](https://github.com/spf13/viper) for configuration. The generated gateway code includes a config file that can be overridden with cli flags or environment variables. For environment-variable overrides, use a `<SERVICE>_` prefix, upcase the setting, and replace `.` with `_`.
```

Comment:
does everything deserve a hyphen :)? what was wrong with `environment variable overrides`?

Reply:
> 

The first two words form an adjective.  See: https://www.grammarly.com/blog/hyphen/